docs/data/joy/components/drawer/drawer.md

```diff
@@ -1,70 +1,88 @@
 ---
 productId: joy-ui
 title: React Drawer component
-components: Drawer
+components: Drawer, ModalClose, DialogContent
```

Comment:
@danilo-leal GH wouldn't let me comment on these lines up here because they weren't part of the PR before, so I went ahead and pushed this change so you can see how to add component links to the API section! 🤝

Reply:
Sweet, that's perfect! I didn't know it was that simple 😅